### PR TITLE
refactor(component): adapt the questionnaire component to the desktop client

### PR DESCRIPTION
**Pull request by JimmFly**
close TOV-240

Windows desktop:
image

MacOS desktop:
image

Web:
image

### CODE REVIEW

**Comment by JimmFly** [comment]
Current dependencies on/for this PR:
* `canary`
  * **PR #5514** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5514?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5514?utm_source=stack-comment).

File: packages/frontend/component/src/components/auth-components/onboarding-page.tsx [hunk 1]
```diff
@@ -104,6 +122,17 @@ export const OnboardingPage = ({
...
   );
+  const isMacosDesktop = environment.isDesktop && environment.isMacOs;
+  const isWindowsDesktop = environment.isDesktop && environment.isWindows;
+
+  useEffect(() => {
+    if (environment.isDesktop) {
+      // to hide macOS window control buttons
+      apis?.ui.handleSidebarVisibilityChange(false).catch(err => {
+        console.error(err);
+      });
```
Comment (joooye34) [inline packages/frontend/component/src/components/auth-components/onboarding-page.tsx:133]: Could we recovery after onboarding is completed?

File: packages/frontend/component/src/components/auth-components/onboarding-page.tsx [hunk 2]
```diff
@@ -50,43 +51,60 @@ function getCallbackUrl(location: Location) {
...
+      </ScrollableContainer>
+      <footer className={styles.footer}>
+        <div className={styles.linkGroup}>
+          <a
+            className={styles.link}
+            href="https://affine.pro/terms"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Terms of Conditions
```
Comment (joooye34) [inline packages/frontend/component/src/components/auth-components/onboarding-page.tsx:83]: i18n

File: packages/frontend/component/src/components/auth-components/onboarding-page.tsx [hunk 3]
```diff
@@ -50,43 +51,60 @@ function getCallbackUrl(location: Location) {
...
+          </a>
+          <Divider orientation="vertical" />
+          <a
+            className={styles.link}
+            href="https://affine.pro/privacy"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Privacy Policy
```
Comment (joooye34) [inline packages/frontend/component/src/components/auth-components/onboarding-page.tsx:92]: i18n

File: packages/frontend/component/src/components/auth-components/onboarding-page.tsx [hunk 4]
```diff
@@ -119,7 +148,25 @@ export const OnboardingPage = ({
...
+            {isWindowsDesktop ? windowControl : null}
+            <Button
+              className={clsx(styles.button, {
+                [styles.disableButton]: questionIdx === 0,
+                [styles.windowsAppButton]: isWindowsDesktop,
+              })}
+              size="extraLarge"
+              onClick={() => setQuestionIdx(questions.length)}
+            >
+              Skip
```
Comment (joooye34) [inline packages/frontend/component/src/components/auth-components/onboarding-page.tsx:163]: i18n

File: packages/frontend/component/src/components/auth-components/onboarding-page.tsx (hunk 1 above)
Comment (JimmFly) [reply packages/frontend/component/src/components/auth-components/onboarding-page.tsx:133]: This window action button will appear whenever the sidebar is displayed.

**Review by joooye34 — APPROVED** [review]

**Pull request merged**